tutordiscovery/patches/local-docker-compose-jobs-services

```diff
@@ -4,4 +4,4 @@ discovery-job:
     DEFAULT_PARTNER_CODE: openedx
   volumes:
     - ../plugins/discovery/apps/settings/tutor:/openedx/discovery/course_discovery/settings/tutor:ro
-  depends_on: {{ [("lms", RUN_LMS), ("mysql", RUN_MYSQL)]|list_if }}
+  depends_on: {{ [("mysql", RUN_MYSQL)]|list_if }}
```

Comment:
`RUN_LMS` does not exist anymore but the discovery service still depends on the lms container. (same comment below)